src/components/toolbox/table/index.js

```diff
@@ -46,7 +46,12 @@ const Table = ({
             Container: Link,
             to: getRowLink(row),
           })}
-          className={grid.row}
+          onClick={onRowClick && onRowClick.bind(null, row)}
```

Comment:
can this validation be part in a different part and avoid to bind the function in here?

Reply:
Discussed with @osvaldovega and there's no better way, so keeping it.